packages/frontend-web/src/types.ts

```diff
@@ -30,9 +30,9 @@ export type IConfirmationAction = 'approve' | 'defer' | 'highlight' | 'reject'|
 
 export type ICommentAction = 'highlight' | 'approve' | 'defer' | 'reject' | 'tag';
 
+export type IModerationAction = 'approve' | 'defer' | 'highlight' | 'reject';
```

Comment:
Could you add a comment about what these correspond to? (something in the UI? or something sent from the backend to the frontend?) Likewise for other constants here.

Reply:
Added some comments.